GOVERNANCE.md

```diff
@@ -0,0 +1,114 @@
+# Governance
+
+This document is designed to document and clarify the governance of the Pulsar (Pulsar-Edit) organization and teams in an open and transparent way.
+The intent is not for this to be a rigid set of rules but this does allow us to have some written reference as to how things are handled when it comes to the team organization and roles.
+
+Members may belong to more than one team or role and some come hand-in-hand with each other.
+
+We do not work on a strict set of responsibilities with a strict governance model, we mostly resemble the [Do-ocracy](https://communitywiki.org/wiki/DoOcracy) approach to governance without too much in the way of formal or elaborate governance and policies but we do of course have to have some guidelines and agreed conventions to have a working organization.
```

Comment:
👍 Do-ocracy sounds about right, that is indeed how we've been doing it at least.